Client/Application/NavigationRouter.swift

```diff
@@ -5,27 +5,15 @@
 import Foundation
 import Shared
 
-// An enum to route to HomePanels
-enum HomePanelPath: String {
-    case bookmarks
-    case topsites
-    case history
-}
-
 // Used by the App to navigate to different views.
 // To open a URL use /open-url or to open a blank tab use /open-url with no params
-enum DeepLink {
-    case homePanel(HomePanelPath)
+enum DeepLink: Equatable {
     init?(urlString: String) {
         let paths = urlString.split(separator: "/")
         guard let component = paths[safe: 0], let componentPath = paths[safe: 1] else {
             return nil
         }
-        if component == "homepanel", let link = HomePanelPath(rawValue: String(componentPath)) {
-            self = .homePanel(link)
-        } else {
-            return nil
-        }
+        return nil
```

Comment:
😆 this init should be removed, if it only ever returns `nil`. Or is this being retained for some future ticket item?

Reply:
It's technically an empty enum. Kept it there just in case we ever wanted to add any deep-linking past opening URLs. This init method will always return nil until we add options to the enum lol.

Still want it removed?